### PR TITLE
feature/deploy

### PR DESCRIPTION
**Pull request by wgermancutraro**
## Summary

I created a heroku account, the postgre db and i deployed the api.

https://gc-express-training.herokuapp.com/health

## Known Issues

-

## Trello Card

https://trello.com/c/iknW7xkz/15-hacer-deploy-con-heroku

### CODE REVIEW

File: app/models/index.js
Comment (martinremis) [inline app/models/index.js]: This can be done in config files, there is no need to make this explicit here

Comment (martinremis) [reply app/models/index.js]: Also, do not access an env var without the config file (it works just fine, but you may "lose" it in the code)

**Review by martinremis — CHANGES_REQUESTED** [review]

File: app/models/index.js
Comment (wgermancutraro) [reply app/models/index.js]: I tried to create a user, password and put the heroku host but with all of that fails. I'll Keep trying!

**Review by martinremis — APPROVED** [review]

**Comment by martinremis** [comment]
Same than #17, remember to wait until #17 merge and then change the base

**Pull request merged**
